src/Pact/Types/Info.hs

```diff
@@ -112,6 +115,28 @@ renderParsed (Parsed d _) = case d of
   (Columns c _) -> "<interactive>:0:" ++ show c
   (Tab _ c _ ) -> "<interactive>:0:" ++ show c
 
+-- | Lame parsing of 'renderInfo' parsing for UX only.
+parseRenderedInfo :: AP.Parser Info
+parseRenderedInfo = peekChar >>= \c -> case c of
+  Nothing -> return (Info Nothing)
+  Just _ -> do
+    fOrI <- takeTill isColon
+    colon'
+    ln <- round <$> scientific
+    colon'
+    cn <- round <$> scientific
+    let delt = case fOrI of
+          "<interactive>"
+            | ln == 0 -> Columns cn 0
+            | otherwise -> Lines (pred ln) cn 0 0
```

Comment:
why `pred ln` here?

Reply:
The heuristic here (which is worth documenting) is "`Lines` if `ln` > `0` otherwise `Columns`". Note then that `Lines` gets a `succ` above, so this matches that. Probably could use a unit test too.